sympy/matrices/matrices.py

```diff
@@ -4366,7 +4368,11 @@ def norm(self, ord=None):
 
         # Matrix Norms
         else:
-            if ord == 2:  # Spectral Norm
+            if ord == 1:  # Maximum column sum
+                def maxcol(x) : return sum(self.applyfunc(lambda y : abs(y)).col(x))
+                return max(list(map(maxcol, range(self.cols))))
```

Comment:
`max() ` can directly take generators and therefore `map()` as a parameter. You can simplify your code to

```
return max(map(maxcol, range(self.cols)))
```